Core/GCTestCase.h

```diff
@@ -79,6 +79,9 @@
 @interface GCMultipleCommitsRepositoryTests : GCMultipleCommitsRepositoryTestCase
 @end
 
+@interface GCUniqueCommitCollectionTests : GCMultipleCommitsRepositoryTestCase
```

Comment:
You don't need a new top-level test class. You only need a category on `GCMultipleCommitsRepositoryTestCase`.